src/applications/personalization/profile/tests/e2e/military-information/military-information-api-error.cypress.spec.js

```diff
@@ -51,3 +51,73 @@ describe('Bad Address Alert - Profile page', () => {
     MilitaryInformation.serviceDownMessageShouldExist();
   });
 });
+
+describe('Military Information - NonVeteran', () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      generateFeatureToggles({
+        profileShowBadAddressIndicator: true,
+        profileAlwaysShowDirectDepositDisplay: true,
+      }),
+    );
+    cy.intercept('GET', '/v0/user', nonVeteranUser);
+
+    cy.intercept('GET', '/v0/profile/full_name', fullName.success);
+    cy.intercept('GET', '/v0/profile/personal_information', resp => {
+      const req = {
+        query: { now: true },
+      };
+      const res = {
+        json: resp.reply,
+      };
+      handleGetPersonalInformationRoute(req, res);
+    });
+    cy.intercept('GET', '/v0/profile/service_history', resp => {
+      return resp.reply(500, serviceHistory.generateServiceHistoryError('403'));
+    });
+  });
+  it('should display non veteran user error on military information page', () => {
+    cy.login(nonVeteranUser);
+    MilitaryInformation.visitMilitaryInformationPage();
+    cy.injectAxeThenAxeCheck();
+    MilitaryInformation.heroErrorMessageShouldNotExist();
+    MilitaryInformation.notAVeteranMessageShouldExist();
+  });
+});
+
+describe('Military Information - NotInDeers', () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      generateFeatureToggles({
+        profileShowBadAddressIndicator: true,
+        profileAlwaysShowDirectDepositDisplay: true,
+      }),
+    );
+    cy.intercept('GET', '/v0/user', user72Success);
+
+    cy.intercept('GET', '/v0/profile/full_name', fullName.success);
+    cy.intercept('GET', '/v0/profile/personal_information', resp => {
+      const req = {
+        query: { now: true },
+      };
+      const res = {
+        json: resp.reply,
+      };
+      handleGetPersonalInformationRoute(req, res);
+    });
+    cy.intercept('GET', '/v0/profile/service_history', resp => {
+      return resp.reply(200, serviceHistory.generateServiceHistoryError('403'));
+    });
+  });
+  it('should display non veteran user error on military information page', () => {
```

Comment:
need to update `it('...')` to have a different message, it is the same as the previous test right now